go-console/.conf/createDockerComposeProduction.ps1

```diff
@@ -0,0 +1,134 @@
+# args are needed
```

Comment:
This file can be removed, this is automatically copied from scripts to the project folder during the creation.